pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+markers =
+    automated: mark a test as an automated and runnable on CI.
```

Comment:
newline

Reply:
is it better to mark some tests `local` ? it looks like majority (and it should be this way) of test we run automatically

Reply:
good idea, I've added marker `manual` and removed `automated`.